src/libasr/runtime/lfortran_intrinsics.c

```diff
@@ -860,20 +860,27 @@ LFORTRAN_API char* _lfortran_strrepeat_c(char* s, int32_t n)
     return dest_char;
 }
 
-// idx starts from 1
-LFORTRAN_API char* _lfortran_str_item(char* s, int32_t idx) {
+// idx1 and idx2 both start from 1
+LFORTRAN_API char* _lfortran_str_copy(char* s, int32_t idx1, int32_t idx2) {
```

Comment:
Needs to be reverted https://github.com/lcompilers/lpython/pull/1538/commits/2f45421c06c2bee6a9918ae71ca0b88530ebd6fd